frontend/src/pages/ActivityReport/Pages/components/GoalPicker.js

```diff
@@ -105,10 +111,50 @@ const GoalPicker = ({
       const topics = await getTopics();
       setTopicOptions(topics);
     }
-
     fetchTopics();
   }, []);
 
+  // Fetch citations for the goal if the source is CLASS or RANs.
+  useEffect(() => {
+    async function fetchCitations() {
+      // If its a monitoring goal and the source is CLASS or RANs, fetch the citations.
+      if (goalForEditing && goalForEditing.source === 'Federal monitoring issues, including CLASS and RANs') {
```

Comment:
any goal could have this source, right? can we use it like this?

Reply:
updated to standard.